### ISSUE TITLE
Error aggregating SRS collection

### ISSUE THREAD

**Issue by jonescc** [aodn/aws-wps#123]
### To reproduce 

Submit

[request.txt](https://github.com/aodn/aws-wps/files/1439277/request.txt)

### What happens

Get the following error during aggregation

```
2017-11-02 23:30:47.091 INFO 1 --- [ main] au.org.emii.aggregator.NetcdfAggregator : Adding /tmp/1b56d936-178d-436d-a344-55c9fe6667d620170311032000-ABOM-L3S_GHRSST-SSTskin-AVHRR_D-1d_day.nc to output file
2017-11-02 23:30:47.355 INFO 1 --- [pool-1-thread-3] a.o.e.download.ParallelDownloadManager : Downloaded http://data.aodn.org.au/IMOS/SRS/SST/ghrsst/L3S-1d/day/2017/20170727032000-ABOM-L3S_GHRSST-SSTskin-AVHRR_D-1d_day.nc
au.org.emii.aggregator.exception.AggregationException: java.io.IOException: -101: NetCDF: HDF error
at au.org.emii.aggregator.NetcdfAggregator.chunkedCopy(NetcdfAggregator.java:249)
at au.org.emii.aggregator.NetcdfAggregator.appendTimeSlices(NetcdfAggregator.java:221)
at au.org.emii.aggregator.NetcdfAggregator.add(NetcdfAggregator.java:111)
at au.org.emii.aggregationworker.AggregationRunner.run(AggregationRunner.java:181)
at org.springframework.boot.SpringApplication.callRunner(SpringApplication.java:732)
at org.springframework.boot.SpringApplication.callRunners(SpringApplication.java:716)
at org.springframework.boot.SpringApplication.afterRefresh(SpringApplication.java:703)
at org.springframework.boot.SpringApplication.run(SpringApplication.java:304)
at org.springframework.boot.SpringApplication.run(SpringApplication.java:1118)
at org.springframework.boot.SpringApplication.run(SpringApplication.java:1107)
at au.org.emii.aggregationworker.Application.main(Application.java:12)
Suppressed: java.io.IOException: -101: NetCDF: HDF error
at ucar.nc2.jni.netcdf.Nc4Iosp.flush(Nc4Iosp.java:3292)
at ucar.nc2.NetcdfFileWriter.flush(NetcdfFileWriter.java:1013)
at ucar.nc2.NetcdfFileWriter.close(NetcdfFileWriter.java:1024)
at au.org.emii.aggregator.NetcdfAggregator.close(NetcdfAggregator.java:120)
at au.org.emii.aggregationworker.AggregationRunner.run(AggregationRunner.java:245)
... 7 more
Caused by: java.io.IOException: -101: NetCDF: HDF error
at ucar.nc2.jni.netcdf.Nc4Iosp.flush(Nc4Iosp.java:3292)
at ucar.nc2.NetcdfFileWriter.flush(NetcdfFileWriter.java:1013)
at au.org.emii.aggregator.NetcdfAggregator.chunkedCopy(NetcdfAggregator.java:245)
... 10 more
```

### What I expect to happen 

Aggregation finishes successfully.

**Comment by jonescc** [comment]
Suspect this is because the aggregation ran out of space for the output file.   We don't seem to be using a custom AMI?

**Comment by jonescc** [comment]
I think we are building the ami, but we aren't using it yet.

**Comment by jonescc** [comment]
Fixed by #157, #159, #121, #122
